src/app/utils/accountPageUtils.js

```diff
@@ -135,6 +135,9 @@ const manipulateAccountPage = (
       td.appendChild(button);
       el.appendChild(td);
     });
+    const freezeCells = el.querySelectorAll('.patFuncFreeze');
+    if (freezeCells) freezeCells.forEach(cell => cell.remove());
+
```

Comment:
I'm probably missing something, but what prevents this from removing a non-empty `.patFuncFreeze` TD?

Reply:
The query selector is for `<input>`s. So, if there is no checkbox, that TD does not end up being selected. The alternative would be to select elements based on the classnames used for the `<td>`s with checkboxes. Any thoughts?

Reply:
Oh. I might have misread your comment. We want to remove all `.patFuncFreeze` `<td>`s

Reply:
You want to remove all `.patFuncFreeze` TDs found in any context? From the PR description, it sounds like you're only seeking to remove `.patFuncFreeze` TDs that don't contain INPUTs (i.e. remove TDs that are empty). I don't see where you're checking for the presence/absence of INPUTs, so it seems like this code would remove the "freeze" functionality for items in all statuses. Is that your intention? Or have I missed where you're checking for INPUTs/item status?

Reply:
On line 101 the INPUTs are selected and then iterated over on line 110. We are removing `.patFuncFreeze` and adding new cells with the button.

Reply:
I see. You remove instances of TD.patFuncFreeze earlier in the course of removing the INPUTs (line 130), so when there are no INPUTs to begin with, you need to do a final cleanup pass to ensure there are no lingering TD.patFuncFreeze's. Thanks!